share/spice/coderwall/coderwall.css

```diff
@@ -0,0 +1,8 @@
+.zci--coderwall a {
```

Comment:
Let's make this selector more specific:

``css
zci--coderwall .c-icon__sub a {
    ...
}
```
```